File: Aprendendo-C/Exercício/menor_de_tres.c
#include <stdio.h>
#include "gabsLib.h"

int main(){

    int a, b, c = 1;

    printf("Primeiro valor: ");
    scanf("%i", &a);

    printf("Segundo valor: ");
    scanf("%i", &b);

    printf("Terceiro valor: ");
    scanf("%i", &c);

    if(a < b && a < c){
        printf("Menor: %i", a);
    }
    else if(b < a && b < c){
        printf("Menor: %i",b);
    }else{
        printf("Menor: %i ",c);
    }

    jumpln();
    return 0;
}

File: Aprendendo-Python/Exercícos/menor_de_tres.py
x = int(input("Primeiro valor: "))
y = int(input("Segundo valor: "))
z = int(input("Terceiro valor: "))

if (x < y and x < z):
    print("Menor: ", x)
elif(y < x):
    print("Menor: ", y)
else:
    print("Menor: ",z)
